crowbar_framework/app/models/service_object.rb

```diff
@@ -176,15 +176,22 @@ def set_to_applying(nodes, inst, pre_cached_nodes)
     end
   end
 
+  def restore_node_to_ready(node)
+    node.crowbar['state'] = 'ready'
+    node.crowbar['state_owner'] = ""
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)